Userland/Libraries/LibJS/Bytecode/ASTCodegen.cpp

```diff
@@ -1757,6 +1754,8 @@ Bytecode::CodeGenerationErrorOr<void> TaggedTemplateLiteral::generate_bytecode(B
     auto strings_reg = generator.allocate_register();
     generator.emit<Bytecode::Op::Store>(strings_reg);
 
+    // FIXME: In theory we only need to record the first register and the length
+    // FIXME: Add support for argument spreading
```

Comment:
can you even spread stuff in a tagged template literal?

Reply:
I do not even know what a tagged template literal is...

Reply:
Well this makes it easier then, I guess